### DISCUSSION TITLE
Mastodon SendinBlue SMTP not working

### DISCUSSION THREAD

**Discussion by dellee69** [mastodon/mastodon#24310] [Q&A]
I have spent the past 3 days trying to get Mastodon to send emails with sendinblue with zero luck. Nothing has worked. sendinble says my account is setup and working fine, Hetzner (my host) says all ports are open.Nothing is working. I have went through Google for days seeking a fix to this. I installed this from the same tutorial some months back and it worked fine. I have installed this 5 times from beginning to end and it never sends test emails on set up or after installation is completed.

![mail-isues](https://user-images.githubusercontent.com/77635237/228334529-a0895674-f42f-46af-bfbb-7e1bc222e869.jpg)

**Comment by DejavuMoe** [comment]
Preferably paste your SMTP configuration environment variable **(please desensitize your username and password)**. Check whether your SMTP needs to be configured with STARTTLS, SSL or TLS, depending on your SMTP service provider, refer to [Docs - SMTP](https://docs.joinmastodon.org/admin/config/#smtp)

**Reply by dellee69** [reply]
Thanks... 

I got it working. I use Hetzner hosting and after repeatedly rebuilding this specific server and installing, it wouldn't work. After days of this, lol, I decided to create an identical server on the same Hetzner account and installed it, worked perfectly first try. I'm still baffled at what was causing it. I even tried installing it on that server one last time, nope, same thing.
